schema/reportdb/postgres/start.sql

```diff
@@ -10,5 +10,4 @@
 -- in this software or its documentation.
 --
 
-create extension dblink;
 
```

Comment:
Since we want to or rather need to keep this file and after this it is empty, maybe put back the original string `-- This file is currently intentionally left blank.`. See [original commit](https://github.com/uyuni-project/uyuni/commit/8229286bf733c07a9d313110b6eb2e8e6c6349bc)

And the same for spacewalk's start.sql